packages/radix/src/components/Code.tsx

```diff
@@ -16,10 +16,13 @@ export const Code = React.forwardRef<HTMLElement, CodeProps>((props, forwardedRe
             boxShadow: `inset 0 0 0 1px ${theme.colors.gray300}`,
             color: theme.colors.gray800,
             fontFamily: theme.fonts.mono,
-            fontSize: '80%',
-            marginX: '0.1em',
-            paddingX: '0.45em',
-            paddingY: '0.1em',
+            fontSize: '85%',
```

Comment:
Seeing it with a fresh eye, it's a bit too small at 80%, especially when in `13px` paragraphs and tables. Upping it a bit to 85%.